docs/gno-infrastructure/setting-up-a-local-chain.md

```diff
@@ -10,8 +10,59 @@ In this tutorial, you will learn how to start a local Gno node (and chain!).
 Additionally, you will see the different options you can use to make your Gno instance unique.
 
 ## Prerequisites
+- **Git**
+- **`make` (for running Makefiles)**
+- **Go 1.21+**
+- **Go Environment Setup**:
+  - Make sure `$GOPATH` is well-defined, and `$GOPATH/bin` is added to your `$PATH` variable.
 
-- [`gnoland` installed](local-setup.md#3-installing-other-gno-tools).
+## Installation
```

Comment:
Just a note that this `gnoland` install section should go in the adequate `gnoland` documentation

Reply:
As soon as the `gnoland` tutorial is out, I replace this section with the link to it.